omnigibson/action_primitives/starter_semantic_action_primitives.py

```diff
@@ -1560,15 +1553,16 @@ def _draw_plan(self, plan):
         trav_map = self.env.scene._trav_map
         for q in plan:
             # The below code is useful for plotting the RRT tree.
-            SEARCHED.append(np.flip(trav_map.world_to_map((q[0], q[1]))))
+            map_point = trav_map.world_to_map((q[0], q[1]))
+            SEARCHED.append(th.flip(map_point, dims=tuple(range(map_point.dim()))))
 
             fig = plt.figure()
             plt.imshow(trav_map.floor_map[0])
             plt.scatter(*zip(*SEARCHED), 5)
             fig.canvas.draw()
 
             # Convert the canvas to image
-            img = np.fromstring(fig.canvas.tostring_rgb(), dtype=np.uint8, sep="")
+            img = th.frombuffer(fig.canvas.tostring_rgb(), dtype=th.uint8)
```

Comment:
hmm I am not familiar with many torch functions here. did you confirm these code works?